src/WebformHelper.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+namespace Drupal\os2forms_rest_api;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\Url;
+use Drupal\key_auth\Authentication\Provider\KeyAuth;
+use Drupal\webform\WebformInterface;
+use Drupal\webform\WebformSubmissionInterface;
+
+/**
+ * Webform helper for helping with webforms.
+ */
+class WebformHelper {
+  use StringTranslationTrait;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  private EntityTypeManagerInterface $entityTypeManager;
+
+  /**
+   * The current user manager.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  private AccountProxyInterface $currentUser;
+
+  /**
+   * The key authentication service.
+   *
+   * @var \Drupal\key_auth\Authentication\Provider\KeyAuth
+   */
+  private KeyAuth $keyAuth;
+
+  /**
+   * Constructor.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager, AccountProxyInterface $currentUser, KeyAuth $keyAuth) {
+    $this->entityTypeManager = $entityTypeManager;
+    $this->currentUser = $currentUser;
+    $this->keyAuth = $keyAuth;
+  }
+
+  /**
+   * Implements hook_webform_third_party_settings_form_alter().
+   *
+   * @phpstan-param array<string, mixed> $form
+   */
+  public function webformThirdPartySettingsFormAlter(array &$form, FormStateInterface $form_state): void {
+    /** @var \Drupal\Core\Entity\EntityForm $formObject */
+    $formObject = $form_state->getFormObject();
+    /** @var \Drupal\webform\WebformInterface $webform */
+    $webform = $formObject->getEntity();
+    $settings = $webform->getThirdPartySetting('os2forms', 'os2forms_rest_api');
+
+    $form['third_party_settings']['os2forms']['os2forms_rest_api'] = [
+      '#type' => 'details',
+      '#open' => TRUE,
+      '#title' => 'REST API',
+      '#tree' => TRUE,
+    ];
+
+    $allowedUsers = $this->loadUsers($settings['allowed_users'] ?? []);
```

Comment:
can it be replaced with function `public function getAllowedUsers`?

Reply:
It can and should be. Fixed in https://github.com/OS2Forms/os2forms_rest_api/pull/1/commits/902d619d66d35033a4d67520ad40deee819b449a.